build/mvn

```diff
@@ -72,7 +72,7 @@ install_mvn() {
   local MVN_VERSION="3.3.3"
 
   install_app \
-    "https://www.apache.org/dyn/closer.lua?action=download&filename=/maven/maven-3/${MVN_VERSION}/binaries" \
```

Comment:
Interestingly enough 3.3.9 works through the URL but 3.3.3 doesn't appear to anymore - I'm not sure if maybe just one of the mirrors is screwed up or if this is a more intentional change.